### ISSUE TITLE
The code is now out of date with newer versions of scipy

### ISSUE THREAD

**Issue by arokem** [uw-biomedical-ml/mri2mri#9]
```
Traceback (most recent call last):
  File "/Users/arokem/miniconda3/envs/mri2mri/bin/mri2mri", line 33, in <module>
    sys.exit(load_entry_point('mri2mri', 'console_scripts', 'mri2mri')())
  File "/Users/arokem/source/mri2mri/mri2mri/mri2mri.py", line 35, in main
    model.set_input({'A': _toTensor(data[:,:,i])})
  File "/Users/arokem/source/mri2mri/mri2mri/mri2mri.py", line 11, in _toTensor
    img = scipy.misc.toimage(nibImg).convert('RGB')
AttributeError: module 'scipy.misc' has no attribute 'toimage'
```

**Comment by arokem** [comment]
Might be resolved by replacing this line https://github.com/uw-biomedical-ml/mri2mri/blob/ae370b228695c6a8eaf5754a727df978f750e265/mri2mri/mri2mri.py#L11

With something like: 

```
img = np.zeros_like(nibImg + (3,))
img += data 
```

**Comment by arokem** [comment]
Closed through #11
